sonar-ws/src/main/gen-java/org/sonarqube/ws/QualityProfiles.java

```diff
@@ -0,0 +1,2483 @@
+        }
+
+        // Construct using org.sonarqube.ws.QualityProfiles.WsSearchResponse.QualityProfile.newBuilder()
+        private Builder() {
+          maybeForceBuilderInitialization();
+        }
+
+        private Builder(
+            com.google.protobuf.GeneratedMessage.BuilderParent parent) {
+          super(parent);
+          maybeForceBuilderInitialization();
+        }
+        private void maybeForceBuilderInitialization() {
+          if (com.google.protobuf.GeneratedMessage.alwaysUseFieldBuilders) {
+          }
+        }
+        private static Builder create() {
+          return new Builder();
+        }
+
+        public Builder clear() {
+          super.clear();
+          key_ = "";
+          bitField0_ = (bitField0_ & ~0x00000001);
+          name_ = "";
+          bitField0_ = (bitField0_ & ~0x00000002);
+          language_ = "";
+          bitField0_ = (bitField0_ & ~0x00000004);
+          languageName_ = "";
+          bitField0_ = (bitField0_ & ~0x00000008);
+          isInherited_ = false;
+          bitField0_ = (bitField0_ & ~0x00000010);
+          parentKey_ = "";
+          bitField0_ = (bitField0_ & ~0x00000020);
+          parentName_ = "";
+          bitField0_ = (bitField0_ & ~0x00000040);
+          isDefault_ = false;
+          bitField0_ = (bitField0_ & ~0x00000080);
+          activeRuleCount_ = 0L;
+          bitField0_ = (bitField0_ & ~0x00000100);
+          projectCount_ = 0L;
+          bitField0_ = (bitField0_ & ~0x00000200);
+          rulesUpdatedAt_ = "";
+          bitField0_ = (bitField0_ & ~0x00000400);
+          return this;
+        }
+
+        public Builder clone() {
+          return create().mergeFrom(buildPartial());
+        }
+
+        public com.google.protobuf.Descriptors.Descriptor
+            getDescriptorForType() {
+          return org.sonarqube.ws.QualityProfiles.internal_static_sonarqube_ws_rules_WsSearchResponse_QualityProfile_descriptor;
+        }
+
+        public org.sonarqube.ws.QualityProfiles.WsSearchResponse.QualityProfile getDefaultInstanceForType() {
+          return org.sonarqube.ws.QualityProfiles.WsSearchResponse.QualityProfile.getDefaultInstance();
+        }
+
+        public org.sonarqube.ws.QualityProfiles.WsSearchResponse.QualityProfile build() {
+          org.sonarqube.ws.QualityProfiles.WsSearchResponse.QualityProfile result = buildPartial();
+          if (!result.isInitialized()) {
+            throw newUninitializedMessageException(result);
+          }
+          return result;
+        }
+
+        public org.sonarqube.ws.QualityProfiles.WsSearchResponse.QualityProfile buildPartial() {
+          org.sonarqube.ws.QualityProfiles.WsSearchResponse.QualityProfile result = new org.sonarqube.ws.QualityProfiles.WsSearchResponse.QualityProfile(this);
+          int from_bitField0_ = bitField0_;
+          int to_bitField0_ = 0;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename this local variable name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS00117)